pymc4/inference/utils.py

```diff
@@ -38,13 +39,15 @@ def trace_to_arviz(pm4_trace, pm4_sample_stats):
     Parameters
     ----------
     pm4_trace : dict
+    pm4_sample_stats : dict
 
     Returns
     -------
     arviz.data.inference_data.InferenceData
     """
-    import arviz as az
 
-    posterior = {k: np.swapaxes(v.numpy(), 1, 0) for k, v in pm4_trace.items()}
+    posterior = {
+        k.split("/")[1]: np.swapaxes(v.numpy(), 1, 0) for k, v in pm4_trace.items() if "/" in k
```

Comment:
This split wouldn't be safe for variables in nested models. Those would have their names as: `outer_model/inner_model/var_name`. Why don't we leave the name with the entire scope? Does arviz complain about the slash characters?

Reply:
yeah, I was not sure this was a good idea :-) I just wanted to remove it because I think is kind of redundant information. We can leave it. Or we can do something like str.split('/', 1)[1].
ArviZ does not complaint about slash characters

Reply:
If arviz doesn't complain about the slash, I would just leave the entire name for now. The full name is needed when one wants to fix values in forward sampling